iset.mm

```diff
@@ -65603,6 +65603,112 @@ elements or fails to hold (is equal to ` (/) ` ) for some element.
+          ZEJNAVSJRZWAUJUJZDOZFRZDUBZWCWEVSIUAZWFUCUAZQZDFUDZWHWEWLPPQZWEWLWMUJ
+          ZWLUKZPSQZUJZWQUKWEWPWOSPUEUFUGUHWEPWLPSUIZQZWNWQULZWEVTPWRAWDWAUMWEW
+          DWRTRVTWRQAWDWAUNWEWLPSTPTRWEUOUHSTRWEUPUHAWDWLUQZWAADFGIJVSLURUSZUTC
+          VSCOZIUAZWJQZDFUDZPSUIWRJTHXCVSQZXFWLPSXGXEWKDFXGXDWIWJXCVSIVAVBVCVDM
+          VEVFVGWEXAWSWTVHXBWLPPSVIVJVLVKVMWKDFVNVJWGWBDBDBFVOVPVQVR $.
+      $}
+
+      ${
+        $d A u v y z $.  $d B z $.  $d F v y z $.  $d O v w $.  $d O v y z $.
+        $d P w $.  $d ph u v y z $.
+        fodjuomni.1 $e |- ( ph -> A. w e. O ( P ` w ) = 1o ) $.
+        $( Lemma for ~ fodjuomni .  The case where A is empty.  (Contributed by
+           Jim Kingdon, 27-Jul-2022.) $)
+        fodjuomnilem0 $p |- ( ph -> A = (/) ) $=
+          ( cv wcel c0 c1o wceq cfv com vu vv wa cinl cdju djulcl foelrn syl2an
+          wfo wrex cif simprl peano1 a1i 1onn wdc fodjuomnilemdc ad2ant2r fveq2
+          ifcldcd eqeq1d rexbidv ifbid fvmptg syl2anc wral simplr simprr eqcomd
+          ad2antrr rspcdva eqeq2d rspcev iftrued 3eqtr3rd rexlimddv wn pm2.65da
+          1n0 nesymi eq0rdv ) AUAEAUANZEOZPQRZAWCUCZWBUDSZUBNZHSZRZWDUBIAIEFUEZ
+          HUIWFWJOWIUBIUJWCKEFWBUFUBIWJWFHUGUHWEWGIOZWIUCZUCZWGGSZWHCNZUDSZRZCE
+          UJZPQUKZQPWMWKWSTOWNWSRWEWKWIULZWMWRPQTPTOWMUMUNQTOWMUOUNAWKWRUPWCWIA
+          CEFHIWGKUQURUTBWGBNZHSZWPRZCEUJZPQUKWSITGXAWGRZXDWRPQXEXCWQCEXEXBWHWP
+          XAWGHUSVAVBVCLVDVEWMDNZGSZQRZWNQRDIWGXFWGRXGWNQXFWGGUSVAAXHDIVFWCWLMV
+          JWTVKWMWRPQWMWCWHWFRZWRAWCWLVGWMWFWHWEWKWIVHVIWQXICWBEWOWBRWPWFWHWOWB
+          UDUSVLVMVEVNVOVPWDVQWEQPVSVTUNVRWA $.
+      $}
+
+      ${
+        $d A v x z $.  $d A v y z $.  $d B z $.  $d F y z $.  $d O f w $.
+        $d O v w y z $.  $d P f w $.  $d P v w y z $.  $d ph v y z $.
+        $( Lemma for ~ fodjuomni .  The final result with ` P ` broken out into
+           a hypothesis.  (Contributed by Jim Kingdon, 29-Jul-2022.) $)
+        fodjuomnilemres $p |- ( ph -> ( E. x x e. A \/ A = (/) ) ) $=
+          ( vw vf vv cv cfv c0 wceq adantr wrex c1o wral wo wcel wex cmap fveq1
+          co eqeq1d rexbidv ralbidv orbi12d comni isomnimap mpbid fodjuomnilemf
+          c2o wb syl rspcdva wa cdju simpr fveq2 cbvrexv sylib fodjuomnilemm ex
+          wfo fodjuomnilem0 orim12d mpd ) AMPZGQZRSZMIUAZVOUBSZMIUCZUDZBPEUEBUF
+          ZERSZUDAVNNPZQZRSZMIUAZWDUBSZMIUCZUDZVTNURIUGUIZGWCGSZWFVQWHVSWKWEVPM
+          IWKWDVORVNWCGUHZUJUKWKWGVRMIWKWDVOUBWLUJULUMAIUNUEZWINWJUCZJAWMWMWNUS
+          JMINUNUOUTUPACDEFGHIJKLUQVAAVQWAVSWBAVQWAAVQVBZBCDOEFGHIAWMVQJTAIEFVC
+          HVJZVQKTLWOVQOPZGQZRSZOIUAAVQVDVPWSMOIVNWQSVOWRRVNWQGVEUJVFVGVHVIAVSW
+          BAVSVBCDMEFGHIAWMVSJTAWPVSKTLAVSVDVKVIVLVM $.
+      $}
+    $}
+
+    ${
+      $d A a b x y z $.  $d B z $.  $d F a b y z $.  $d O a y z $.
+      $d ph y z $.
+      $( A condition which ensures ` A ` is either inhabited or empty.  Lemma
+         3.2 of [PradicBrown2021], p. 4.  (Contributed by Jim Kingdon,
```

Comment:
Can you phrase that condition ? E.g.: If..., then A is either inhabited of empty.

Reply:
I didn't write that, it is a quote from the paper.

My philosophy is that the formal statement of the theorem should (most of the time) be how we state what is being proved. Especially when proposed wording doesn't end up being any more concise or otherwise different from the formal notation.

Reply:
If you wrote the statement in plain English above (the quotation from the paper), it is because it helps.  So we both think it is helpful. Putting it in the comment in iset.mm would therefore be helpful.